pinot-core/src/main/java/org/apache/pinot/core/query/executor/ServerQueryExecutorV1Impl.java

```diff
@@ -167,10 +167,7 @@ public DataTable processQuery(ServerQueryRequest queryRequest, ExecutorService e
     // TODO: Change broker to watch both IdealState and ExternalView to not query the removed segments
```

Comment:
This `TODO` is already done. You may remove this comment block

Reply:
Removed todo.